packages/core/src/Stores/common-store.js

```diff
@@ -40,6 +40,8 @@ export default class CommonStore extends BaseStore {
     @observable app_routing_history = [];
     @observable app_router = { history: null };
 
+    @observable should_skip_raise_error = false;
```

Comment:
can we call it something else? for example `should_skip_login_error` so it's clearer what this prop is for. also should it be in UI store instead, since it affects the UI?

Reply:
How about `should_skip_invalid_token_error` as it's just for invalid tokens and not for all login failures? The reason for adding this to **common-store** is the logic for showing general errors is there (through `setError`)